wolfcrypt/src/random.c

```diff
@@ -1038,8 +1043,14 @@ static void Entropy_StopThread(void)
     #error "ENTROPY_NUM_64BIT_WORDS must be <= SHA3-256 digest size in bytes"
 #endif
 
+#if ENTROPY_BLOCK_SZ < ENTROPY_NUM_UPDATES_BITS
```

Comment:
Double check possible divide by zero:

Setting the `ENTROPY_NUM_UPDATES_BITS` could end up with a division by zero:
`CFLAGS="-DENTROPY_NUM_WORDS_BITS=8 -DENTROPY_NUM_UPDATES=8"`

```
wolfcrypt/src/random.c: In function 'Entropy_MemUse':
wolfcrypt/src/random.c:1037:23: error: division by zero [-Werror=div-by-zero]
1037 |     (ENTROPY_BLOCK_SZ / ENTROPY_NUM_UPDATES_BITS)
```

Reply:
Is it possible to document a bit more about how to configure wolfEntropy?

It seems to be controlled by these defines:
* ENTROPY_NUM_UPDATES, default 18
* ENTROPY_NUM_UPDATES_BITS, depends on ENTROPY_NUM_UPDATES:  /* Upper round of log2(ENTROPY_NUM_UPDATES) */
* ENTROPY_NUM_WORDS_BITS, range [8 … 30], default 14
 ```

Any hints/suggestions that we should take into consideration when choosing the values for the above?

Reply:
Added check for when ENTROPY_NUM_UPDATES_BITS is 0